docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.md

```diff
@@ -13,6 +13,12 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Spryker has more than one thousand modules, and a bunch of them uses in each Spryker-based project. At the same time, the Sprykers team implements about 50 module updates each month. In this case, keeping your project up to date and receiving all necessary updates, coud take a lot of time for the project team.
+
+To improve the developer experience and decrease the time to keep your project up to date - we implemented the Upgrader and Integrator tools. These tools can automatically update vendors of the spryker-based projects and integrate changes to the project side.
+
+For correct updating a project by Upgrader and Integrator, the project has to follow some Spryker upgradability rules. You can check that your project code is compliant with the rules by the Evaluator tool.
```

Comment:
```
If you want to reduce the amount of manual work that is required in update process, your project must follow the specified <better name of these rules should be here> rules.
```
We can't say ```upgradability rules```. We should have better name for that, something like "architectural rules" or "Spryker extendability rules" or something like that

Reply:
Thanks 

Replaced with the suggested structure 
I chose `Spryker extendability rules` - it looks fine from my perspective